build.zig

```diff
@@ -158,6 +158,12 @@ pub fn build(b: *std.Build) !void {
         "Build a Position Independent Executable. Default true for system packages.",
     ) orelse system_package;
 
+    const strip = b.option(
+        bool,
+        "strip",
+        "Build the website data for the website.",
```

Comment:
I think this might have been a copy-and-pasted description. 😉

Reply:
woops, I edited it locally but forgot to stage it :grimacing: